lib/_stream_readable.js

```diff
@@ -758,8 +758,10 @@ Readable.prototype.unpipe = function(dest) {
     state.pipesCount = 0;
     state.flowing = false;
 
-    for (var i = 0; i < len; i++)
+    for (var i = 0; i < len; i++) {
+      unpipeInfo = { hasUnpiped: false };
       dests[i].emit('unpipe', this, unpipeInfo);
```

Comment:
Tiny nit: I'd just create the new object here, without using the above assignment.

Reply:
There are three situations to use unpipeInfo. The above assignment can be used in two situations. I’m also fine to assign it just before using it.